src/js/views/eva-study-view.js

```diff
@@ -48,7 +48,7 @@ EvaStudyView.prototype = {
                 } catch (e) {
                     console.log(e);
                 }
-                _this._parseData();
+                // _this._parseData();
```

Comment:
Please remove code instead of commenting it, or document the comments if they have a very good reason to stay.